python-stdlib/time/time.py

```diff
@@ -0,0 +1,54 @@
+import re
+
+_TS_YEAR = const(0)
+_TS_MON = const(1)
+_TS_MDAY = const(2)
+_TS_HOUR = const(3)
+_TS_MIN = const(4)
+_TS_SEC = const(5)
+_TS_WDAY = const(6)
+_TS_YDAY = const(7)
+_TS_ISDST = const(8)
+
+_WDAY = const(("Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday", "Sunday"))
+_MDAY = const(
+    (
+        "January",
+        "February",
+        "March",
+        "April",
+        "May",
+        "June",
+        "July",
+        "August",
+        "September",
+        "October",
+        "November",
+        "December",
+    )
+)
+
+_time_regex = None
+
+
+def strftime(datefmt, ts):
+    global _time_regex
+    if _time_regex is None:
+        _time_regex = re.compile(r"%(-?[a-zA-Z])")
+    return _time_regex.sub(r"%(\1)s", datefmt) % {
+        "a": _WDAY[ts[_TS_WDAY] - 1][0:3],
```

Comment:
I think wday is 0-6 for Mon-Sun, so there's no need to do the `-1` here and below?